### ISSUE TITLE
linear-classify incorrect matrix multiplication

### ISSUE THREAD

**Issue by mgruben** [cs231n/cs231n.github.io#120]
Under the `Interpreting a linear classifier` section, the `ship score` should be `60.75`.

Shown below are the simplification steps (it appears that the bias was not subtracted in the original maths, see step 5.)
```
1. <0, 0.25, 0.2, -0.3> * <56, 231, 24, 2> + (-1.2)
2. 0*56 + 0.25*231 + 0.2*24 + (-0.3)*2 - 1.2
3. 0 + 57.75 + 4.8 - 0.6 - 1.2
4. 62.55 - 0.6 - 1.2
5. 61.95 - 1.2
6. 60.75

```

**Comment by harirajv** [comment]
Yes, the ship score should have been 60.75. Instead 61.95 is given which is the value without adding bias.

<img width="844" alt="Screenshot 2021-06-25 at 10 13 59 AM" src="https://user-images.githubusercontent.com/31141010/123370858-37a45e00-d59e-11eb-979f-0df19d1551bc.png">

**Comment by harirajv** [comment]
@ranjaykrishna Shall I create a similar image with the proper values?
